packages/styles/variables.css

```diff
@@ -69,7 +69,7 @@
   --space-small: 16px;
   --space-medium: 18px;
   --space-large: 24px;
-  --space-largest: 29px;
+  --space-largest: 32px;
```

Comment:
I'm not sure if this constitutes a breaking change or not, and I could be convinced either direction but I think warrants discussion.

Reply:
Yeah, this is the one that jumped out at me. I _think_ it's fine from a breaking change perspective. In some ways, it's no worse than other changes we make.

Reply:
Who needs to discuss it? This is what Bobby said the largest space should be.

Reply:
I'm not sure, it just means anything utilizing this var outside of cauldron would shift. I'm okay with changing it, just trying to figure out the impact. It's only used in a few spots in the extension/walnut so the impact would be minimal there and we will detect any visual changes via the extension screenshots.